extra_docs/quickstart.md

```diff
@@ -128,58 +140,68 @@ If you watch closely, the API is chainable, so you can rewrite the query in a mu
 
 ``` Javascript
 [..]
-jexiaClient(fetch).init({projectID: "your Jexia App URL", key: "username", secret: "password"}, dataModule);
+jexiaClient(fetch).init(credentials, dataModule);
 
 dataModule.dataset("posts")
   .select()
   .execute()
-  .then( (records) => {
+  .then((records) => {
     // you can start iterating through the posts here
-  }).catch( (error) => {
+  }).catch((error) => {
     // there was a problem retrieving the records
   });
 [..]
 ```
 
 But at the very least, defining dataset variables could come in handy when executing multiple different queries on the same dataset.
 
-### Sorting records
+### [Sorting records](#sorting-records)
 
 To activate sorting, you need to call `.sortAsc` or `.sortDesc` on a `Query` object and pass as string the name of the field you want to sort by.
 
 ``` Javascript
 [..]
-posts.select().sortAsc("id").execute().then( (records) => {
-  // operate on your sorted records here
-});
+posts.select()
+  .sortAsc("id")
+  .execute()
+  .then((records) => {
+    // operate on your sorted records here
+  });
 [..]
 ```
 
-### Limit and offset
+### [Limit and offset](#limit-offset)
 
-You can use `.limit` and `.offset` on a `Query` object for these options. They can be used separately or together. Only setting the limit (to a value X) will make the query operate on the first X records. Only setting the offset will make the query operate on the last Y records, starting from the offset value. 
+You can use `.limit` and `.offset` on a `Query` object for these options. They can be used separately or together. Only setting the limit (to a value X) will make the query operate on the first X records. Only setting the `offset` will make the query operate on the last Y records, starting from the `offset` value.
 
 ``` Javascript
 [..]
-posts.select().limit(2).offset(5).execute().then( (records) => {
-  // records will be an array of 2 records, starting from position 5 in the Dataset
-});
+posts.select()
+  .limit(2)
+  .offset(5)
+  .execute()
+  .then((records) => {
+    // records will be an array of 2 records, starting from position 5 in the Dataset
+  });
 [..]
 ```
 
-### Only retrieving certain fields
+### [Only retrieving certain fields](#retrieve-certain-fields)
 
 You can use the `.fields` method to define what fields you want to retrieve.
 
 ``` Javascript
 [..]
-posts.select().fields("id", "title").execute().then( (records) => {
-  // the individual record objects will only have the two properties defined above
-});
+posts.select()
+  .fields("id", "title")
+  .execute()
+  .then((records) => {
+    // the individual record objects will only have the two properties defined above
+  });
 [..]
 ```
 
-### Aggregation functions
+### [Aggregation functions](#aggregation-functions)
```

Comment:
Backend doesn't support those... Or are you supporting them in SDK code (until backend provides support)?

Reply:
hmm, there is no logic to handle this operations in SDK, so I'd say we should remove this feature or at least turn it off for now.

Reply:
Sorry @marlosin, but there is some logic to handle aggregation in SDK. Do we really need to remove it right now? Or better wait some time until it will be supported on backend?

Reply:
I didn't remove anything, what do you think we remove this section from the guide and then put back when it becomes available again? @KutsenkoA @veger ?